docker/docker-compose.yml

```diff
@@ -28,6 +28,8 @@ services:
       - db
     depends_on:
       - hsqldb
+    volumes:
+      - ../../xmpp-logs:/usr/local/radar/xmpp-server/logs
     environment:
       RADAR_XMPP_FCM_SENDER_KEY: "1043784930865"
```

Comment:
ok to include these?

Reply:
Yes. This is just a fcm dummy test project i use. No real devices here.

Reply:
ok